ansible/library/lag_facts.py

```diff
@@ -70,6 +71,8 @@ def get_po_names(self):
                 self.module.fail_json(msg=fail_msg)
             else:
                 for po in out.split():
+                    # load db config
+                    load_db_config()
```

Comment:
can we do `load_db_config` just once in the beginning, iso doing for every portchannel or every NS?

Reply:
Fixed as suggested.